app/grandchallenge/profiles/views.py

```diff
@@ -197,3 +206,94 @@ def form_invalid(self, form):
         # and display an error message
         messages.add_message(self.request, messages.ERROR, "Incorrect token.")
         return response
+
+
+class SubscriptionUpdate(SuccessMessageMixin, UserPassesTestMixin, UpdateView):
```

Comment:
Should this be subscription update or email preferences? It still keeps people subscribed to notifications and forum tracks, just stops sending them emails.

Reply:
Good point! `EmailPreferencesUpdate` is better!